doc-content/enterprise-only/admin-and-config/viewing-task-variables-basic-filters-in-bc-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='viewing-task-variables-basic-filters-in-bc-proc']
+= Viewing task variables using basic filters
+
+{CENTRAL} provides basic filters to view task variables in *Manage Tasks* and *Task Inbox*. You can view the task variables of the task as columns using *Show/hide columns*.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Tasks* or  go to *Menu* -> *Track* -> *Task Inbox*.
+. On the *Task Inbox* page, click the filter icon on the left of the page to expand the *Filters* panel
```

Comment:
Should the customer be able to choose their method?
If yes, then the next line should be changed to reflect this and the subsequent steps checked to make sure they correctly reflect the page the user is on.

s/On the *Task Inbox* page,/On the *Manage Tasks* page or the *Task Inbox* page,